lib/account/signin.dart

```diff
@@ -1,6 +1,12 @@
 // Essential Flutter packages
+import 'dart:ui';
+import 'package:atlas/inc/include.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
```

Comment:
Order is wrong, check: [Effective Dart style guide: Ordering](https://dart.dev/guides/language/effective-dart/style#ordering)

**Right order: -**

// Dart arms
import 'dart:ui';

// Essential Flutter packages
import 'package:flutter/cupertino.dart';
import 'package:flutter/gestures.dart';
import 'package:flutter/material.dart';

// Internal relatives
import 'inc/include.dart' show Include;